packages/store/addon/-private/caches/identifier-cache.ts

```diff
@@ -41,6 +41,10 @@ if (macroCondition(getOwnConfig<{ polyfillUUID: boolean }>().polyfillUUID)) {
 }
 
 function uuidv4(): string {
+  assert(
```

Comment:
note asserts only trigger in dev build, so if deploying to an insecure environment as mentioned in https://discord.com/channels/480462759797063690/486549196837486592/1040645066504556564 the error will still be simply that `randomUUID` is not a function.

Reply:
(also note, even for internal tools, if you aren't deploying production builds with https you very much should be for a host of reasons)